src/operator/mshadow_op.h

```diff
@@ -127,6 +131,20 @@ MXNET_UNARY_MATH_OP(softsign, a / (1.0f + math::fabs(a)));
 
 MXNET_UNARY_MATH_OP(softsign_grad, 1.0f /  math::sqr(1.0f + math::fabs(a)));
 
+#define gelu_gx(a) \
```

Comment:
define MXNET_GELU_GX

Reply:
Done.

Reply:
Can we have MXNET_ as the prefix for the macro? thanks

Reply:
Done.